SmartPlaces.Facilities/lib/OntologyMapper/src/OntologyMapper.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<Sdk Name="Microsoft.Build.CentralPackageVersions" />
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<Authors>Microsoft</Authors>
+		<Description>Support for converting from one DTDL-based Ontology to a different DTDL Based Ontology</Description>
+		<copyright>© Microsoft Corporation. All rights reserved.</copyright>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Azure.DigitalTwins.Parser" Version="6.3.3-preview" />
```

Comment:
Should we be open sourcing with a preview package?

Reply:
It's the only version that supports DTDLv3